.github/auto-approve.yml

```diff
@@ -0,0 +1,25 @@
+rules:
+# Currently we allow a small subset of valid types of PRs to get merged.  The following three
+# rules can be used altogether or any subset. The bot confirms that incoming pull requests'
+# author match the authors below, and subsequently, the allowed title of the pull requests.
+# `changedFiles` are allowed patterns for the file paths that are changed in a PR, and 
+# `maxFiles` are the maximum amount of files that are allowed to be changed in that PR. Neither
+# changedFiles nor maxFiles are required, but they will be enforced on a PR if they are set.
+# Each rule only has the allowed values listed below; there is no mixing of values between rules,
+# nor can any values be replaced. To understand the full set of rules for the JSON file, check out
+# src/valid-pr-schema.json
+
+# Option 5: incoming PRs from renovate-bot
+- author: "renovate-bot"
+
+# Title for PRs from renovate-bot MUST start with `fix(deps):` or `chore(deps)"`.
+  title: "^(fix\\(deps\\)|chore\\(deps\\)):"
+
+# (Optional) The only changed file paths in an incoming PR MUST end with `package.json`. These are listed in regex. If this property 
+# is omitted, auto-approve bot will allow PRs that match the author and title to merge, without
+# checking the file paths. 
+  changedFiles:
+  - "/pom\\.xml$"
```

Comment:
This is specifically suffix? or should it be `.*/pom.xml$`?

Tangential - this "changedFiles" inclusive or exclusive or the rest of the items in the list?

Reply:
This is specifically a suffix, as (I assume) we don't know how many directory levels deep pom.xml should be. 

For renovate-bot PRs, changedFiles is inclusive - no file in the PR can violate the regex rules of the naming.

Reply:
To clarify - if we put in:
```
- "/pom\\.xml@"
- "/gradle\\.build$"
```

Would it accept PRs that have both pom _and_ gradle files OR would it be both pom _or_ gradle files?

Reply:
Should be AND

Reply:
If it's AND do we want to list then as two separate regexs? Does that mean it'll only approve PRs with gradle AND poms? What happens if it's just poms?

Reply:
@sofisl At some point we should add gradle.build, but that's not updating ATM.
@kurtisvg The app doesn't like gradle.build